atomic_reactor/build.py

```diff
@@ -9,16 +9,11 @@
 Classes which implement tasks which builder has to be capable of doing.
 Logic above these classes has to set the workflow itself.
 """
-import re
```

Comment:
Why changes in build.py were done?

Reply:
in config I need dockerfileimages, which were in InsideBuilder (which will be removed), so I've already moved useful attributes to workflow

        self.df_dir = build_file_dir
        self._df_path = None
        self.original_df = None
        self.buildargs = {}  # --buildargs for container build
        self.dockerfile_images = DockerfileImages([])

Reply:
Why dockerfile images must be in config?

Reply:
they are only used there to set source registry & organization 
https://github.com/containerbuildsystem/atomic-reactor/pull/1654/files#diff-3079e0bf8bc543b512b09e347622e9a2457237692c644efe6319d113c0ef4f1eR185